src/MenuItem/MenuItem.js

```diff
@@ -20,23 +20,25 @@ const MenuItem = React.createClass({
     checked: React.PropTypes.bool,
 
     /**
-     * Elements passed as children to inner ListItem.
+     * Elements passed as children to the underlying `ListItem`.
      */
     children: React.PropTypes.node,
 
     /**
      * @ignore
-     * Indicates if the menu should render with compact desktop styles.
+     * If true, the menu item will render with compact desktop
+     * styles.
      */
     desktop: React.PropTypes.bool,
```

Comment:
If `desktop` is true on `Menu` then the child menu items have this automatically set to true
https://github.com/callemall/material-ui/blob/master/src/Menu/Menu.js#L263